src/mesh/vtk_helper.jl

```diff
@@ -0,0 +1,108 @@
+
+"""
+    n_verts_between(n, from, to, dim)
+
+Compute the coordinates of n equally distributed points between the points
+given by `from` and `to`. `dim` is the dimension of `from` and `to`. 
+Inspired by: https://github.com/ju-kreber/paraview-scripts/blob/master/node_ordering.py
+"""
+function n_verts_between(n, from, to)
+    if n <= 0
+        return
+    end
+    dim = length(from)
+    @assert length(from) == length(to)
+    edge_verts = [LinRange(from[i], to[i], n+2)[2: 1+n] for i in 1:dim]
+    return edge_verts
+end
+
+
+"""
+    triangle_vtk_order(corner_verts, order, dim, skip = false)
+
+Compute the coordinates of a `VTK_LAGRANGE_TRIANGLE` of a triangle or order `order` 
+defined by the coordinates of the vertices given in `corner_verts`. `dim` is the
+dimension of the coordinates given. If `skip` is set to true, the coordinates
+of the vertex- and edge-points aren't computed, which can be used to compute
+points of a `VTK_LAGRANGE_WEDGE`
+Inspired by: https://github.com/ju-kreber/paraview-scripts/blob/master/node_ordering.py
+"""
+function triangle_vtk_order(corner_verts, order, dim, skip = false)
+    if order < 0
+        return nothing
+    end
+    if order == 0
+        # For recursion
+        return corner_verts[:,1]
+    end
+    #Corner vertices
+    coords = Matrix{Float64}(undef, 0, dim)
+    if skip == false
+        coords = corner_verts
+    end
+    #edge vertices
+    num_verts_on_edge = order -1 
+    # edges in vtk-order
+    edges = [(1,2), (2,3), (3,1)]
+    for (frm, to) in edges
+        if skip == false
+            tmp = n_verts_between(num_verts_on_edge, corner_verts[:, frm], corner_verts[:, to])
+            tmp_vec = Vector{Float64}(undef, dim)
+            for i in 1:num_verts_on_edge
+                for j in 1:dim
+                    tmp_vec[j] = tmp[j][i]
+                end
+                coords = hcat(coords, tmp_vec)
+            end
+        end
+    end
+    if order == 2
+        return coords
+    end
+    #faces
+    e_x = (corner_verts[:,2] .- corner_verts[:,1]) ./ order
+    e_y = (corner_verts[:,3] .- corner_verts[:,1]) ./ order
+    inc = [(e_x + e_y) (-2*e_x + e_y) (e_x - 2*e_y)]
+    if order >= 3
+        coords = hcat(coords, triangle_vtk_order(corner_verts .+ inc, order - 3, dim, false))
+    end
+    return coords
+end
+
+"""
+    vtk_order(elem::Tri, order)
+
+Construct all node-points of a VTK_LAGRANGE_TRIANGLE of order order. The corner-nodes are
+given by the reference-triangle used by StartUpDG
+"""
+function vtk_order(elem::Tri, order)
+    tri_sud_vertices = [-1.0 1.0 -1.0; -1.0 -1.0 1.0]
```

Comment:
```suggestion
    tri_sud_vertices = permutedims(hcat(nodes(Tri(), 1)...))
```
This way, if we change the node ordering, the VTK writer will automatically adapt to it. Can you check that this works?